### ISSUE TITLE
Character encoding issue in the generated HTML

### ISSUE THREAD

**Issue by rsicarelli** [PatilShreyas/compose-report-to-html#88]
Hello!

I've encountered a character encoding issue in the HTML output generated. This issue is causing misrepresentation of certain characters in the generated HTML file.

Examples:
```
âŒ Non Skippable âœ… Restartable
âŒ Non Skippable âŒ Non Restartable
âœ… Skippable âœ… Restartable
âŒ Unstable class MyClass
```
Examples:
<img width="227" alt="image" src="https://github.com/PatilShreyas/compose-report-to-html/assets/3150335/794310ca-e78c-4c20-aff1-e8791d07b2fc">
<img width="277" alt="image" src="https://github.com/PatilShreyas/compose-report-to-html/assets/3150335/266f76cc-1aea-4bee-9ce1-edbd50f1ba0a">

Could you please shed some light on how I can overcome this problem?

Thank you!

**Comment by PatilShreyas** [comment]
Actually these are emoticons.

**Comment by PatilShreyas** [comment]
Released in `v1.3.0`
